<!DOCTYPE html>
<html>
  <title>Arun Durvasula</title>
  <head>
		<script async src="https://www.googletagmanager.com/gtag/js?id=G-9T8GZ2LZ70"></script>
		<script>
		  window.dataLayer = window.dataLayer || [];
		  function gtag(){dataLayer.push(arguments);}
		  gtag('js', new Date());

		  gtag('config', 'G-9T8GZ2LZ70');
		</script>
    <style>
    div.content { max-width: 600px; margin: auto; font-family: Arial, Geneva, Helvetica, sans-serif; }
    a {color: grey;}
  </style>
  </head>

	<body>
  <div class="content">
  <h1 style="font-family:arial;">Arun Durvasula, PhD</h1>

My research focuses on developing statistical methods to understand patterns of genetic variation by focusing on two major questions:
	<ol>
    <li>How does evolutionary history shape patterns of genetic variation?</li>
    <li>How do we learn about the biological mechanisms that link genetic variation to phenotypic variation?</li>
  </ol>

	I am a Postdoctoral Research Fellow in the <a href="https://genetics.hms.harvard.edu">Department of Genetics</a> at Harvard Medical School and the <a href="https://heb.fas.harvard.edu">Department of Human Evolutionary Biology</a>, where I am co-advised by <a href="https://reich.hms.harvard.edu">David Reich</a> and <a href="https://www.hsph.harvard.edu/alkes-price/">Alkes Price</a>.

  Previously, I was an <a href="https://github.com/ybrandvain/GRFP/blob/master/Arun_Durvasula_Research_Proposal_2018.pdf">NSF graduate research fellow</a> at UCLA in the <a href="https://medschool.ucla.edu/human-genetics">Department of Human Genetics</a> working with <a href="https://www.eeb.ucla.edu/Faculty/Lohmueller/">Kirk Lohmueller</a> and <a href="http://web.cs.ucla.edu/~sriram/">Sriram Sankararaman</a>.

  I also worked in <a href="http://www.mpipz.mpg.de/hancock">Angela Hancock's lab</a> at the Max Planck Institute for Plant Breeding Research studying the demographic history of <i>Arabidopsis thaliana</i>.

  Prior to that, I worked in <a href="http://rilab.ucdavis.edu">Jeff Ross-Ibarra's lab</a> studying the effects of recent population size expansion on linked selection in maize and developing population genetic analysis tools at UC Davis.


  You can find me on the web in the following places:
  <p style="text-align: center;"><a href="https://github.com/arundurvasula" target="_blank" rel="noopener noreferrer">Github</a> | <a href="https://twitter.com/arundurvasula" target="_blank" rel="noopener noreferrer">Twitter</a> | <a href="https://scholar.google.com/citations?user=ZisO_BUAAAAJ&hl=en&oi=ao" target="_blank" rel="noopener noreferrer">Google Scholar</a> | <a href="pdfs/Arun_Durvasula_CV_Oct2021.pdf">CV</a></p>
  <p style="text-align: center;">Email: [firstname].[lastname]@gmail.com</p>


	<h2 style="font-family:arial;">Software</h2>

	<p> <a href='https://github.com/sriramlab/OrientAGraph'>OrientAGraph</a>: Maximum likelihood network orientation for admixture graphs [<a href="https://academic.oup.com/bioinformatics/article/37/Supplement_1/i142/6319689">12</a>] </p>
	<p> <a href='https://stdpopsim.readthedocs.io/en/latest/'>stdpopsim</a>: Standard library for population genetic simulation models [<a href="https://elifesciences.org/articles/54967">9</a>] </p>
	<p> <a href='https://github.com/sriramlab/archie'>ArchIE</a>: Reference-free inference of archaic local ancestry [<a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008175">7</a>] </p>
	<p> <a href='https://github.com/ANGSD-wrapper/angsd-wrapper'>ANGSD-wrapper</a>: Utilities for analyzing next generation sequencing data [<a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full">2</a>] </p>

  <h2 style="font-family:arial;">Publications</h2>

( + ) Joint first authors
( * ) Joint senior authors

    <hr />

		<p><strong>Preprint(s)</strong></p>

	        <p>Zhang X+, Kim BY+, Singh A, Sankararaman S, <strong>Durvasula A*</strong>, Lohmueller KE*. MaLAdapt reveals novel targets of adaptive introgression from Neanderthals and Denisovans in worldwide human populations. <a href="https://www.biorxiv.org/content/10.1101/2022.05.16.491756v1" style="color:#FF0000">Preprint</a></p>
	  
		<p>Huang B, <strong>Durvasula A</strong>, Mousavi N, Ziaei-Jam H, Maksimov M, Lohmueller KE, Gymrek M. Genome-wide selection inference at short tandem repeats. <a href="https://www.biorxiv.org/content/10.1101/2022.05.12.491726v1" style="color:#FF0000">Preprint</a></p>

		<hr />

		<strong>2021</strong>

		<p>12. Molloy EK, <strong>Durvasula A</strong>, Sankararaman S. Advancing admixture graph estimation via maximum likelihood network orientation. <a href="https://academic.oup.com/bioinformatics/article/37/Supplement_1/i142/6319689" style="color:#FF0000">Bioinformatics (ISMB 2021)</a> (<a title="molloy2021" href="pdfs/molloy2021.pdf">PDF</a>) (<a href="https://github.com/ekmolloy/OrientAGraph">Code</a>)</p>

		<p>11. <strong>Durvasula A</strong>, Lohmueller KE. Negative selection on complex traits limits phenotype prediction accuracy between populations. <a href="https://www.cell.com/ajhg/fulltext/S0002-9297(21)00055-0" style="color:#FF0000">American Journal of Human Genetics</a> (<a title="Durvasula2021" href="pdfs/durvasula2021.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href="https://facultyopinions.com/prime/739707987">F1000</a>]</p>

		<p>10. Boocock J, Sadhu MJ, <strong>Durvasula A</strong>, Bloom JS, Kruglyak L. Ancient balancing selection maintains incompatible versions of a conserved metabolic pathway in yeast. <a href="https://www.science.org/doi/abs/10.1126/science.aba0542" style="color:#FF0000">Science</a> (<a title="Boocock2021" href="pdfs/boocock2021.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://science.sciencemag.org/content/371/6527/358.3'>Science</a>] </p>

		<hr />

		<strong>2020</strong>

		<p>9. The PopSim Consortium. A community-maintained standard library of population genetic models. <a href="https://elifesciences.org/articles/54967" style="color:#FF0000">eLife</a> (<a title="Popsim2020" href="pdfs/popsim2020.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://www.nature.com/articles/s41592-020-0951-4'>Nature Methods</a>] </p>


		<p>8. <strong>Durvasula A</strong>, Sankararaman S. Recovering signals of ghost archaic introgression in African populations <a href="https://www.science.org/doi/full/10.1126/sciadv.aax5097" style="color:#FF0000">Science Advances</a> (<a href="https://drive.google.com/file/d/1Q7iEdjCf7d4HnI65_JzjIYnsDP8G0Iy8/view?usp=sharing">Download local ancestry maps</a>) (<a title="Durvasula2019" href="pdfs/durvasula2020.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://www.nytimes.com/2020/02/12/science/west-africa-ancient-humans.html'>New York Times</a>] [<a href='https://www.npr.org/2020/02/12/805237120/ghost-dna-in-west-africans-complicates-story-of-human-origins'>NPR</a>] [<a href="https://facultyopinions.com/prime/737361910">F1000</a>] [<a href='https://scienceadvances.altmetric.com/details/75930979/news'>Others</a>] </p>

    <hr />

	<strong>2019</strong>

    <p>7. <strong>Durvasula A</strong>, Sankararaman S. A statistical model for reference-free inference of archaic local ancestry. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008175" style="color:#FF0000">PLOS Genetics</a> (<a href="https://github.com/sriramlab/archie">Code</a>) (<a title="Durvasula2019" href="pdfs/durvasula2019.pdf">PDF</a>)</p>

    <hr />

    <strong>2018</strong>

    <p>6. Huber CD+, <strong>Durvasula A+</strong>, Hancock AM, Lohmueller KE. Gene expression drives the evolution of dominance. <a href="https://www.nature.com/articles/s41467-018-05281-7" style="color:#FF0000">Nature Communications</a> (<a href="https://github.com/LohmuellerLab/dominance">Code</a>) (<a title="Huber2018" href="pdfs/huber2018.pdf">PDF</a>)</p>

    <p>5. Schumer M, Xu C, Powell D, <strong>Durvasula A</strong>, et al. Natural selection interacts with the local recombination rate to shape the evolution of hybrid genomes. <a href="https://www.science.org/doi/abs/10.1126/science.aar3684" style="color:#FF0000">Science</a> (<a title="schumer2018" href="pdfs/schumer2018.pdf">PDF</a>)
    <p style="padding-left:30px;">Highlights: [<a href="https://facultyopinions.com/prime/733074082">F1000</a>]</p>

    <p>4. Schweizer RM, <strong>Durvasula A</strong>, et al. Natural selection and origin of a melanistic allele in North American gray wolves. <a href="https://academic.oup.com/mbe/article/35/5/1190/4913657" style="color:#FF0000">Molecular Biology and Evolution</a> (<a title="Schweizer2018" href="pdfs/schweizer2018.pdf">PDF</a>)</p>

    <hr />

    <strong>2017</strong>
    <p>3. <strong>Durvasula A+</strong>, Fulgione A+ , et al. African genomes illuminate the early history and transition to selfing in Arabidopsis thaliana. <a href="http://www.pnas.org/content/114/20/5213" style="color:#FF0000">PNAS</a> (<a href="https://github.com/HancockLab/African_A.thaliana">Code</a>) (<a title="Durvasula2017" href="pdfs/durvasula2017.pdf">PDF</a>)</p>

    <p style="text-align:left;padding-left:30px;"><span class="footnote"><span class="slug-metadata-note ahead-of-print"><span class="slug-doi" title="10.1073/pnas.1616736114">Highlights: [<a href="https://pubmed.ncbi.nlm.nih.gov/28572379/">Science</a>] [<a href="https://www.nature.com/articles/nplants201791">Nature Plants</a>] [<a href="https://facultyopinions.com/prime/727580348">F1000</a>]</span></span><span class="slug-pub-date">
    </span></span></p>


    <hr />

    <strong>2016</strong>

    <p>2. <strong>Durvasula A+</strong>, Hoffman PJ+, et al. ANGSD-wrapper: utilities for analyzing next generation sequencing data. <a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full" style="color:#FF0000">Molecular Ecology Resources</a> (<a href="https://github.com/ANGSD-wrapper/angsd-wrapper">Code</a>) (<a title="Durvasula2016" href="pdfs/durvasula2016.pdf">PDF</a>) </p>

    <p>1. Beissinger TM, Wang L, Crosby K, <strong>Durvasula A</strong>, et al. Recent demography drives changes in linked selection across the maize genome. <a href="http://www.nature.com/articles/nplants201684" style="color:#FF0000">Nature Plants</a> (<a href="https://github.com/timbeissinger/Maize-Teo-Scripts">Code</a>) (<a title="Beissinger2016" href="pdfs/beissinger2016.pdf">PDF</a>)</p>

    <hr />
    <p>Last updated May 2022</p>
  </div>
	</body>
</html>
