<!DOCTYPE html>
<html>
  <title>Arun Durvasula</title>
  <head>
    <style>
    div.content { width: 480px; font-family: Arial, Geneva, Helvetica, sans-serif; }
  </style>
  </head>

	<body>
	<h1 style="font-family:arial;">Arun Durvasula</h1>
  <div class="content">
  I'm Arun Durvasula, a PhD student at UCLA in the <a href="https://www.genetics.ucla.edu/">Department of Human Genetics</a> working with <a href="https://www.eeb.ucla.edu/Faculty/Lohmueller/">Kirk Lohmueller</a> and <a href="http://web.cs.ucla.edu/~sriram/">Sriram Sankararaman</a>. I study the forces that shape and maintain genetic variation.

  Previously, I worked in <a href="http://www.mpipz.mpg.de/hancock">Angela Hancock's lab</a> at the Max Planck Institute for Plant Breeding Research studying demography in <i>Arabidopsis thaliana</i>.

  Prior to that, I worked in <a href="http://rilab.org/" target="_blank" rel="noopener noreferrer">Jeff Ross-Ibarra's lab</a>  studying demography in maize, developing analysis tools, and looking at crop-wild introgression in <em>Oryza</em> at UC Davis.

  I study genetics, statistics, and computer science. You can find me on the web in the following places:
  <p style="text-align: center;"><a href="https://github.com/arundurvasula" target="_blank" rel="noopener noreferrer">Github</a> | <a href="https://twitter.com/arundurvasula" target="_blank" rel="noopener noreferrer">Twitter</a> | <a href="https://scholar.google.com/citations?user=ZisO_BUAAAAJ&hl=en&oi=ao" target="_blank" rel="noopener noreferrer">Google Scholar</a> | <a title="arun-durvasula-cv-july-2018" href="https://arundurvasula.files.wordpress.com/2018/07/arun-durvasula-cv-july-2018.pdf">CV</a></p>
  <p style="text-align: center;">Email: [firstname].[lastname]@gmail.com</p>

  <h2 style="font-family:arial;">Publications</h2>

( + ) denotes equal contribution

    <p><strong>Preprints</strong></p>

    <p><strong>Durvasula A</strong>, Sankararaman S (2019) Recovering signals of ghost archaic introgression in African populations (<a href="https://www.biorxiv.org/content/early/2018/03/21/285734">Preprint</a>) (<a href="https://github.com/sriramlab/ArchIE">Code</a>) (<a href="https://twitter.com/arundurvasula/status/976461771107008514">Tweet</a> , <a href="https://twitter.com/arundurvasula/status/1101541607805870080">Revision</a>)</p>

    <hr />

    <strong>2019</strong>
    <strong>Durvasula A</strong>, Sankararaman S. A statistical model for reference-free inferece of archaic local ancestry. PLoS Genetics. In press.

    <hr />

    <strong>2018</strong>

    <p>Huber CD+, <strong>Durvasula A+</strong>, Hancock AM, Lohmueller KE. Gene expression drives the evolution of dominance. <a href="https://www.nature.com/articles/s41467-018-05281-7">Nature Communications</a>.(<a href="http://www.biorxiv.org/content/early/2017/08/31/182865">Preprint</a>)(<a href="https://github.com/LohmuellerLab/dominance">Code</a>) (<a title="Huber2018" href="pdfs/huber2018.pdf">PDF</a>)</p>

    <p>Schumer M, Xu C, Powell D, <strong>Durvasula A</strong>, Skov L, Holland C, Sankararaman S, Andolfatto P, Rosenthal G, Przeworski M. Natural selection interacts with the local recombination rate to shape the evolution of hybrid genomes. <a href="http://science.sciencemag.org/content/360/6389/656">Science</a>. (<a href="https://www.biorxiv.org/content/early/2017/11/01/212407">Preprint</a>) (<a title="schumer2018" href="pdfs/schumer2018.pdf">PDF</a>)
    <p style="padding-left:30px;">Highlights: [<a href="https://f1000.com/prime/733074082">F1000</a>]</p>

    <p>Schweizer RM, <strong>Durvasula A</strong>, Smith J, Vohr SH, Stahler DR, Galaverni M, Thalmann O, Smith DW, Randi E, Ostrander EA, Green RE, Lohmueller KE, Novembre J,Wayne RK. Natural selection and origin of a melanistic allele in North American gray wolves. <em><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy031/4913657">Molecular Biology and Evolution</a>. </em>(Open Access) (<a title="Schweizer2018" href="pdfs/schweizer2018.pdf">PDF</a>)</p>

    <hr />

    <strong>2017</strong>
    <p style="text-align:left;"><span class="footnote"><strong><span class="slug-metadata-note ahead-of-print"><span class="slug-doi" title="10.1073/pnas.1616736114">Durvasula A+ </span></span></strong><span class="slug-metadata-note ahead-of-print"><span class="slug-doi" title="10.1073/pnas.1616736114">, Fulgione A+ , </span></span></span><span class="name">Gutaker RM</span>,<span class="name"> Alacakaptan SI, </span><span class="name">Flood PJ, </span><span class="name">Neto C</span>, <span class="name">Tsuchimatsu T, </span><span class="name">Burbano HA</span>,<span class="name"> Xavier Picó F, </span><span class="name">Alonso-Blanco C</span>, <span class="name">Hancock AM.</span> African genomes illuminate the early history and transition to selfing in <em>Arabidopsis thaliana. <a href="http://www.pnas.org/content/114/20/5213" target="_blank" rel="noopener noreferrer">Proceedings of the National Academy of Sciences</a>. </em>(Open Access) (<a href="https://github.com/HancockLab/African_A.thaliana">Code</a>) (<a title="Durvasula2017" href="https://arundurvasula.files.wordpress.com/2018/04/durvasula20171.pdf">PDF</a>) (<a href="https://twitter.com/arundurvasula/status/860226982641926144">Tweet</a>)</p>
    <p style="text-align:left;padding-left:30px;"><span class="footnote"><span class="slug-metadata-note ahead-of-print"><span class="slug-doi" title="10.1073/pnas.1616736114">Highlights: [<a href="http://science.sciencemag.org/content/356/6341/919.2">Science</a>] [<a href="https://www.nature.com/articles/nplants201791">Nature Plants</a>] [<a href="https://f1000.com/prime/727580348">F1000</a>]</span></span><span class="slug-pub-date">
    </span></span></p>


    <hr />

    <strong>2016</strong>

    <strong>Durvasula A+</strong>, Hoffman PJ+, Kent TV, Liu C, Kono TJY, Morrell PL, Ross-Ibarra J. ANGSD-wrapper: utilities for analyzing next generation sequencing data. <em><a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full" target="_blank" rel="noopener noreferrer">Molecular Ecology Resources</a></em>. (<a href="https://doi.org/10.7287/peerj.preprints.1472v2">Preprint</a>) (<a href="https://github.com/mojaveazure/angsd-wrapper">Code</a>) (<a title="Durvasula2016" href="https://arundurvasula.files.wordpress.com/2018/04/durvasula2016.pdf">PDF</a>)

    Beissinger TM, Wang L, Crosby K, <strong>Durvasula A</strong>, Hufford MH, Ross-Ibarra J. Recent demography drives changes in linked selection across the maize genome. <em><a href="http://www.nature.com/articles/nplants201684" target="_blank" rel="noopener noreferrer">Nature Plants</a>. </em>(<a href="http://dx.doi.org/10.1101/031666">Preprint</a>) (<a href="https://github.com/timbeissinger/Maize-Teo-Scripts">Code</a>) (<a title="Beissinger2016" href="https://arundurvasula.files.wordpress.com/2018/04/beissinger2016.pdf">PDF</a>)

    <hr />
  </div>
	</body>
</html>
