<!DOCTYPE html>
<html>
  <title>Arun Durvasula</title>
  <head>
    <style>
    div.content { max-width: 600px; margin: auto; font-family: Arial, Geneva, Helvetica, sans-serif; }
    a {color: grey;}
  </style>
  </head>

	<body>
  <div class="content">
  <h1 style="font-family:arial;">Arun Durvasula</h1>

My research focuses on developing statistical methods to understand patterns of genetic variation by focusing on two major questions:
	<ol>
    <li>How does evolutionary history shape patterns of genetic variation?</li>
    <li>How do we learn about the biological mechanisms that link genetic variation to phenotypic variation?</li>
  </ol>

  I am an NSF graduate research fellow at UCLA in the <a href="https://medschool.ucla.edu/human-genetics">Department of Human Genetics</a> working with <a href="https://www.eeb.ucla.edu/Faculty/Lohmueller/">Kirk Lohmueller</a> and <a href="http://web.cs.ucla.edu/~sriram/">Sriram Sankararaman</a>.

  Previously, I worked in <a href="http://www.mpipz.mpg.de/hancock">Angela Hancock's lab</a> at the Max Planck Institute for Plant Breeding Research studying the demographic history of <i>Arabidopsis thaliana</i>.

  Prior to that, I worked in <a href="http://rilab.org/">Jeff Ross-Ibarra's lab</a> studying the effects of recent population size expansion on linked selection in maize and developing population genetic analysis tools at UC Davis.

  I study genetics, statistics, and computer science. You can find me on the web in the following places:
  <p style="text-align: center;"><a href="https://github.com/arundurvasula" target="_blank" rel="noopener noreferrer">Github</a> | <a href="https://twitter.com/arundurvasula" target="_blank" rel="noopener noreferrer">Twitter</a> | <a href="https://scholar.google.com/citations?user=ZisO_BUAAAAJ&hl=en&oi=ao" target="_blank" rel="noopener noreferrer">Google Scholar</a> | <a href="pdfs/Arun_Durvasula_CV_Jan2021.pdf">CV</a></p>
  <p style="text-align: center;">Email: [firstname].[lastname]@gmail.com</p>




  <h2 style="font-family:arial;">Publications</h2>

( + ) denotes equal contribution

    <hr />

		<p><strong>Submitted</strong></p>

		<p>Robles C, Ganna A, Gusev A, <strong>Durvasula A</strong>, Gazal S, Loh PR, Reich DE, Sankararaman S. The impact of Neanderthal admixture on the genetic architecture of complex traits in present-day Europeans.</p>

		<hr />

    <p><strong>Preprint(s)</strong></p>

    <p><strong>Durvasula A</strong>, Lohmueller KE. Negative selection on complex traits limits genetic risk prediction accuracy between populations <a href="https://www.biorxiv.org/content/10.1101/721936v1" style="color:#FF0000">Preprint</a></p>

    <hr />

		<strong>2021</strong>

		<p>10. Boocock J, Sadhu MJ, <strong>Durvasula A</strong>, Bloom JS, Kruglyak L. Ancient balancing selection maintains incompatible versions of a conserved metabolic pathway in yeast. <a href="https://science.sciencemag.org/content/371/6527/415" style="color:#FF0000">Science</a> (<a title="Boocock2021" href="pdfs/boocock2021.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://science.sciencemag.org/content/371/6527/358.3'>Science</a>] </p>

		<hr />

		<strong>2020</strong>

		<p>9. The PopSim Consortium. A community-maintained standard library of population genetic models. <a href="https://elifesciences.org/articles/54967" style="color:#FF0000">eLife</a> (<a title="Popsim2020" href="pdfs/popsim2020.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://www.nature.com/articles/s41592-020-0951-4'>Nature Methods</a>] </p>


		<p>8. <strong>Durvasula A</strong>, Sankararaman S. Recovering signals of ghost archaic introgression in African populations <a href="https://advances.sciencemag.org/content/6/7/eaax5097" style="color:#FF0000">Science Advances</a> (<a href="https://sriramlab.cass.idre.ucla.edu/public/afr_archaic_maps_release.zip">Download local ancestry maps</a>) (<a title="Durvasula2019" href="pdfs/durvasula2020.pdf">PDF</a>)</p>
		<p style="padding-left:30px;">Highlights: [<a href='https://www.nytimes.com/2020/02/12/science/west-africa-ancient-humans.html'>New York Times</a>] [<a href='https://www.npr.org/2020/02/12/805237120/ghost-dna-in-west-africans-complicates-story-of-human-origins'>NPR</a>] [<a href="https://f1000.com/prime/737361910">F1000</a>] [<a href='https://scienceadvances.altmetric.com/details/75930979/news'>Others</a>] </p>

    <hr />

		<strong>2019</strong>

    <p>7. <strong>Durvasula A</strong>, Sankararaman S. A statistical model for reference-free inference of archaic local ancestry. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008175" style="color:#FF0000">PLOS Genetics</a> (<a href="https://github.com/sriramlab/archie">Code</a>) (<a title="Durvasula2019" href="pdfs/durvasula2019.pdf">PDF</a>)</p>

    <hr />

    <strong>2018</strong>

    <p>6. Huber CD+, <strong>Durvasula A+</strong>, Hancock AM, Lohmueller KE. Gene expression drives the evolution of dominance. <a href="https://www.nature.com/articles/s41467-018-05281-7" style="color:#FF0000">Nature Communications</a> (<a href="https://github.com/LohmuellerLab/dominance">Code</a>) (<a title="Huber2018" href="pdfs/huber2018.pdf">PDF</a>)</p>

    <p>5. Schumer M, Xu C, Powell D, <strong>Durvasula A</strong>, et al. Natural selection interacts with the local recombination rate to shape the evolution of hybrid genomes. <a href="http://science.sciencemag.org/content/360/6389/656" style="color:#FF0000">Science</a> (<a title="schumer2018" href="pdfs/schumer2018.pdf">PDF</a>)
    <p style="padding-left:30px;">Highlights: [<a href="https://f1000.com/prime/733074082">F1000</a>]</p>

    <p>4. Schweizer RM, <strong>Durvasula A</strong>, et al. Natural selection and origin of a melanistic allele in North American gray wolves. <a href="https://academic.oup.com/mbe/article/35/5/1190/4913657" style="color:#FF0000">MBE</a> (<a title="Schweizer2018" href="pdfs/schweizer2018.pdf">PDF</a>)</p>

    <hr />

    <strong>2017</strong>
    <p>3. <strong>Durvasula A+</strong>, Fulgione A+ , et al. African genomes illuminate the early history and transition to selfing in Arabidopsis thaliana. <a href="http://www.pnas.org/content/114/20/5213" style="color:#FF0000">PNAS</a> (<a href="https://github.com/HancockLab/African_A.thaliana">Code</a>) (<a title="Durvasula2017" href="pdfs/durvasula2017.pdf">PDF</a>)</p>

    <p style="text-align:left;padding-left:30px;"><span class="footnote"><span class="slug-metadata-note ahead-of-print"><span class="slug-doi" title="10.1073/pnas.1616736114">Highlights: [<a href="http://science.sciencemag.org/content/356/6341/919.2">Science</a>] [<a href="https://www.nature.com/articles/nplants201791">Nature Plants</a>] [<a href="https://f1000.com/prime/727580348">F1000</a>]</span></span><span class="slug-pub-date">
    </span></span></p>


    <hr />

    <strong>2016</strong>

    <p>2. <strong>Durvasula A+</strong>, Hoffman PJ+, et al. ANGSD-wrapper: utilities for analyzing next generation sequencing data. <a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full" style="color:#FF0000">Mol Eco Res</a> (<a href="https://github.com/mojaveazure/angsd-wrapper">Code</a>) (<a title="Durvasula2016" href="pdfs/durvasula2016.pdf">PDF</a>) </p>

    <p>1. Beissinger TM, Wang L, Crosby K, <strong>Durvasula A</strong>, et al. Recent demography drives changes in linked selection across the maize genome. <a href="http://www.nature.com/articles/nplants201684" style="color:#FF0000">Nature Plants</a> (<a href="https://github.com/timbeissinger/Maize-Teo-Scripts">Code</a>) (<a title="Beissinger2016" href="pdfs/beissinger2016.pdf">PDF</a>)</p>

    <hr />
    <p>Last updated January 2021</p>
  </div>
	</body>
</html>
